### PR TITLE
Nextcloud

### PR DESCRIPTION
**Pull request by Serph91P**
Added Redis to next cloud for caching purposes

### CODE REVIEW

File: servapps/Nextcloud/cosmos-compose.json [hunk 1]
```diff
@@ -110,7 +112,7 @@
     },
     
     "{ServiceName}-mariadb": {
-      "image": "mariadb:10.11",
+      "image": "mariadb:latest",
```
Comment (Serph91P) [inline servapps/Nextcloud/cosmos-compose.json:115]: Any Reason you did not use the latest version? Never had Problems on my previous setup

File: servapps/Nextcloud/cosmos-compose.json [hunk 2]
```diff
@@ -138,6 +140,33 @@
...
+      "volumes": [
+        {
+          "source": "{ServiceName}-redis-data",
+          "target": "/bitnami/redis",
+          "type": "volume"
+        }
+      ],
+      "environment": [
+        "ALLOW_EMPTY_PASSWORD=no",
+        "REDIS_PASSWORD={Passwords.2}",
```
Comment (Serph91P) [inline servapps/Nextcloud/cosmos-compose.json:167]: Hope this is the way to use a generated password.

**Comment by azukaar** [comment]
Same here this has SAB in it ^^

**Comment by Serph91P** [comment]
> Same here this has SAB in it ^^

oh shoot, yeah will clean this out and reupload sorry :)

**Pull request closed without merging**